### PR TITLE
Fixes#256 Change Fossasia samples to FOSSASIA Summit

### PR DESCRIPTION
**Pull request by ParthS007**
In reference to https://github.com/fossasia/open-event-webapp/issues/1610

### CODE REVIEW

**Comment by ParthS007** [comment]
@mariobehling @SaptakS @shubham-padia @geekyd

File: sample/FOSSASIASummit2017/GoogleIO17/config.json [hunk 1]
```diff
@@ -0,0 +1,6 @@
+{
```
Comment (geekyd) [inline sample/FOSSASIASummit2017/GoogleIO17/config.json]: @ParthS007 Incorrect path for GoogleIO2017. It should not be inside fossaissummit2017 directory

Comment (ParthS007) [reply sample/FOSSASIASummit2017/GoogleIO17/config.json]: @geekyd  I have not changed it, It is same as before

Comment (ParthS007) [reply sample/FOSSASIASummit2017/GoogleIO17/config.json]: Ohh, by mistake.

**Comment by ParthS007** [comment]
@geekyd  where I have to do the change?

**Comment by ParthS007** [comment]
@geekyd  I have done it already 
image

**Comment by ParthS007** [comment]
Please tell If any place is left to be changed

**Comment by geekyd** [comment]
@ParthS007 Yes. Looks good to me. After this is merged you need to update the links of the events for travis in the open event web app.

**Comment by ParthS007** [comment]
yes @geekyd , I will do it

**Pull request merged**